### ISSUE TITLE
Installing on windows

### ISSUE THREAD

**Issue by mrgloom** [wkentaro/labelme#36]
For windows this
```
conda create --name=labelme python=2.7
source activate labelme
conda install pyqt
pip install labelme
```

should be (I have anaconda python 3.6)
```
conda install pyqt
cd labelme-master
conda create --name=labelme python=3.6
activate labelme
pip install labelme
```

For some reason I can install `conda install pyqt` only outside of env, otherwise I get error:
```
CondaEnvironmentNotFoundError: Could not find environment: C:\Users\myuser\Anaco
nda3\envs\labelme .
You can list all discoverable environments with `conda info --envs`.
```

**Comment by mrgloom** [comment]
When I click tutorial in GUI I get error:

```
Traceback (most recent call last):
  File "c:\users\myuser\anaconda3\lib\site-packages\labelme\app.py", line 447, i
n tutorial
    subprocess.Popen([self.screencastViewer, self.screencast])
  File "c:\users\myuser\anaconda3\lib\subprocess.py", line 707, in __init__
    restore_signals, start_new_session)
  File "c:\users\myuser\anaconda3\lib\subprocess.py", line 990, in _execute_chil
d
    startupinfo)
FileNotFoundError: [WinError 2] Не удается найти указанный файл
```

**Comment by mrgloom** [comment]
Also when I try to save markup data I get this error:
![err](https://user-images.githubusercontent.com/4003908/32948993-3645d8f8-cbb2-11e7-9566-e020864f0577.PNG)

**Comment by mrgloom** [comment]
Seems work on python 2 via `C:\Users\myuser\Anaconda2\Scripts\pip.exe install labelme`
But at first run show message:
```
External WM_DESTROY received for  QWidgetWindow(0x7237890, name="LabelDialogClas
sWindow") , parent:  QWindow(0x0) , transient parent:  QWidgetWindow(0x7205040,
name="MainWindowClassWindow")
```

And also fail on Help->Tutorial
```
Traceback (most recent call last):
  File "c:\users\myuser\anaconda2\lib\site-packages\labelme\app.py", line 447, i
n tutorial
    subprocess.Popen([self.screencastViewer, self.screencast])
  File "c:\users\myuser\anaconda2\lib\subprocess.py", line 390, in __init__
    errread, errwrite)
  File "c:\users\myuser\anaconda2\lib\subprocess.py", line 640, in _execute_chil
d
    startupinfo)
WindowsError: [Error 2]
```

**Comment by KatsuhiroMorishita** [comment]
>Also when I try to save markup data I get this error:
me too

on Windows 10, Anaconda 4.4.0 Python 3.6.1

**Comment by wkentaro** [comment]
It seems that this issue is same as https://github.com/wkentaro/labelme/issues/25 and should be fixed via #61.
Please try again if you are still interested.
